app/models/person.rb

```diff
@@ -169,6 +169,60 @@ def remote_ticket?
     ticket.present? and ticket.remote_ticket_id.present?
   end
 
+  def merge_with(doppelgaenger, keep_last_updated=false)
+    keep = self
+    kill = doppelgaenger
+
+    keep, kill = kill, keep if keep_last_updated and self.updated_at < doppelgaenger.updated_at
+
```

Comment:
I find the first part up to this comment confusing. Why not add a class method `def Person.combine(keep, kill)`, move all the following code there and do something like

~~~ruby
return Person.combine(doppelgaenger, self) if keep_last_updated && newer?(doppelgaenger)
Person.combine(self, doppelgaenger) 
~~~

Extracting the `updated_at` check into a method is just for documentation of course.

(I kind of regret using explicit `self` in most of frabs code, since you only need it in assignments. Most code bases omit it.)